lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -417,3 +415,41 @@ export class PublicClientApplication implements IPublicClientApplication {
         return this.controller.clearCache(logoutRequest);
     }
 }
+
+/**
+ * creates NestedAppAuthController and passes it to the PublicClientApplication,
+ * falls back to StandardController if NestedAppAuthController is not available
+ *
+ * @param configuration
+ * @returns IPublicClientApplication
+ *
+ */
+export async function createNestablePublicClientApplication(
+    configuration: Configuration
+): Promise<IPublicClientApplication> {
+    const nestedAppAuth = new NestedAppOperatingContext(configuration);
+    await nestedAppAuth.initialize();
+
+    if (nestedAppAuth.isAvailable()) {
+        const controller = new NestedAppAuthController(nestedAppAuth);
+        return new PublicClientApplication(configuration, controller);
```

Comment:
Do we need to call initialize on this here too before returning?

Reply:
We should. Without initialize the `nestedApp.isAvailable` is not populated.

Reply:
Sorry, I'm asking about pca.initialize, which I don't see here.

Reply:
For NAA, in `NestedAppAuthController` the initialize does nothing. It is an empty function added for the sake of consistency with the other controllers. If we see a future need for it, we can add now, but it will be an empty function.  I can check if it affects functionality, but I don't think it does.

Reply:
I would add the init call _just in case_ someone adds some logic to NAAController in the future and we forget to call it here...